crates/l2/monitor/widget/node_status.rs

```diff
@@ -53,7 +63,11 @@ impl NodeStatusTable {
                 "Last Known Block:".to_string(),
                 last_known_block.to_string(),
             ),
-            ("Peers:".to_string(), follower_nodes.to_string()),
+            if is_based {
```

Comment:
why is this needed? In general it seems an antipattern to have a `is_based` flag spread across the codebase.

Reply:
I agree, part of the problem is due to the fact that the table rows are rendered (read rows elements are formatted into `String`) in `refresh_items` instead of in the `SatefulWidget` implementation, where we have the state of the widget that knows whether we are on based or not, and can render the Peers row or not, reducing all the flag spread thought this code to a single `if`.

We could either file an issue or tackle this in this PR, what do you think?

Reply:
it's ok to add a ticket and fix later

Reply:
cc @iovoid file an issue explaining this and link this thread.